traffic_monitor/experimental/traffic_monitor/enum/enum.go

```diff
@@ -44,6 +43,8 @@ type CacheType string
 const (
 	// CacheTypeEdge represents an edge cache.
 	CacheTypeEdge = CacheType("EDGE")
+	// CacheTypeEdgeVector represents a vector (teak) cache.
+	CacheTypeEdgeVector = CacheType("EDGE_VECTOR")
```

Comment:
This has to be removed before it can be merged, Vector is implementation-specific.